src/Libraries/Microsoft.Extensions.AI.Evaluation.Quality/Utilities/ParsingUtilities.cs

```diff
@@ -76,4 +78,76 @@ await chatConfig.ChatClient.GetResponseAsync(
 
         return response.Text.Trim();
     }
+
+    internal static bool TryParseNumericScore(
```

Comment:
nit: It might be better to name this something like `TryParseTaggedResponseText` or something similar, since there are other forms of numeric output that we parse as well.

Reply:
Sure - this one is internal, but happy to rename it. Some of the new evaluators from the python SDK (which we haven't added yet) also use the same format for Boolean metrics - so perhaps we will need to say something like `TryParseTaggedResponseTextWithNumericScore` :)